deployments/helm/k8s-dra-driver/values.yaml

```diff
@@ -35,6 +35,7 @@ selectorLabelsOverride: {}
 allowDefaultNamespace: false
 
 deviceClasses: ["gpu", "mig", "imex"]
+maskNvidiaDriverParams: false
```

Comment:
do we want to add a description to the value to call out that this is for testing under kind with multiple nodes?

Reply:
updated

Reply:
I would assume that a user would first inspect the values file to determine what can be set and not necessarily go as deep as the template file.

I would summarise it as follows: the comment in the template file is for us, and the comment in the values file is for users.

> What is your preference?

I prefer grouping options for readability, but I can also see that doing this early is not ideal -- especially if we don't have something that makes sense as a logical unit. Let's leave it top-level for now.

Reply:
Updated the comment in the template to be more geared towards us rather than the user.